src/http_client/client.rs

```diff
@@ -0,0 +1,59 @@
+use std::{str::FromStr, sync::Arc};
+
+use async_rustls::rustls::{OwnedTrustAnchor, RootCertStore};
+use hyper::Uri;
+
+use super::{
+    async_std_compat::{self, CompatConnector},
+    request::RequestBuilder,
+};
+
+type HyperClient = Arc<hyper::Client<CompatConnector, hyper::Body>>;
+
+#[derive(Clone)]
+pub struct Client {
+    pub(crate) hyper: HyperClient,
+}
+
+impl Client {
+    pub fn new() -> Self {
+        ClientBuilder::default().build()
+    }
+
+    pub fn get(&self, uri: impl AsRef<str>) -> RequestBuilder {
```

Comment:
is there reason for another layout of abstraction?  Maybe this shouldn't be called `Client`,  Something like `ReadOnlyClient` or specific purpose

Reply:
I removed the `ClientBuilder` layer and moved its contents to `Client` itself